tests/vcr_python_wrapper.py

```diff
@@ -107,6 +107,31 @@ def filter_response(response):
         # headers should be case insensitive, but for some reason they weren't for me
         response['headers'].pop(header.lower(), None)
         response['headers'].pop(header, None)
+    try:
+        json_body = json.loads(response['body']['string'])
+    except json.JSONDecodeError:
+        pass
+    else:
+        if "host" in json_body:
+            json_body['host'] = 'FILTERED'
+        if "account" in json_body:
+            json_body['account'] = 'FILTERED'
+        if "base_dn" in json_body:
+            json_body['base_dn'] = 'FILTERED'
+        if "groups_base" in json_body:
+            json_body['groups_base'] = 'FILTERED'
+        # another for the search result
+        if 'results' in json_body:
+            if len(json_body["results"]) == 1:
+                if "host" in json_body['results'][0]:
```

Comment:
Still seems to fail on this line, but i don't see why.

Reply:
Maybe there is always a result, but empty?